modules/extensions/autoscaler.tf

```diff
@@ -152,3 +153,17 @@ resource "null_resource" "cluster_autoscaler" {
     inline = ["kubectl apply -f ${local.cluster_autoscaler_manifest_path}"]
   }
 }
+
+resource "kubernetes_config_map_v1" "autoscaler_expander_config_map" {
```

Comment:
I think this could be terraformed in our code outside of the module. We want to try to avoid making too many changes here so hopefully we can unfork eventually and this is pretty specific to our usage